app/build.gradle

```diff
@@ -273,7 +273,7 @@ dependencies {
   implementation "androidx.camera:camera-core:$versions.camerax"
```

Comment:
The commit message is correct but it has multiple changes and it's hard to understand which change is for which plugin or dependency

Reply:
Oh, those are all for the plugin. Before the plugin update, the dependency issue was not visible in logs, only after updating to AGP 4.1.0 it started showing up. That's why I combined it in this commit instead of a separate one.